src/plugins/regions.ts

```diff
@@ -61,7 +61,7 @@ export type RegionParams = {
 
 class SingleRegion extends EventEmitter<RegionEvents> {
   public element: HTMLElement
-  public id: string
+  public id: string | string[]
```

Comment:
I would keep this as a string because you're always joining it into a string if it's an array, so it's never actually an array.

Reply:
This is needed to parse an input array, so this will have to be kept in unless I'm missing something (which I very well might!)

Reply:
Update: I just committed a change that makes region.id to be an string array, rather than a string with multiple id's separated by a space. I thought it would be more intuitive on the front end than splitting a string. So this should make these changes necessary

Reply:
IMHO it's better to stick to one type of value, especially since it's called `id` and not `ids`.
I would prefer to avoid all the `if`s down the line. I suggest that `id` property stays a string.